core/templates/dev/head/pages/collection_editor/collection_editor.html

```diff
@@ -89,6 +85,7 @@
 
   <script src="{{TEMPLATE_DIR_PREFIX}}/domain/collection/CollectionNodeObjectFactory.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/domain/collection/CollectionObjectFactory.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/domain/collection/CollectionRightsObjectFactory.js"></script>
```

Comment:
This should go under the ``CollectionRightsBackendApiService`` file; we try to enforce alphabetical sorting when possible.